samples/system-test/spanner.test.js

```diff
@@ -41,277 +41,298 @@ const spanner = new Spanner({
   projectId: PROJECT_ID,
 });
 
-test.before(tools.checkCredentials);
-
-test.before(async () => {
-  const instance = spanner.instance(INSTANCE_ID);
-  const database = instance.database(DATABASE_ID);
-
-  try {
-    await instance.delete();
-  } catch (err) {
-    // Ignore error
-  }
-
-  try {
-    await database.delete();
-  } catch (err) {
-    // Ignore error
-  }
-
-  const [, operation] = await instance.create({
-    config: 'regional-us-central1',
-    nodes: 1,
-    labels: {
-      'gcloud-sample-tests': 'true',
-    },
+describe('Spanner', () => {
+  before(() => tools.checkCredentials);
```

Comment:
I don't know if mocha supports multiple before statements like this.  Can you just pull this line down into the `before(...` statement below?

Reply:
Mocha support multiple before. But if you like then I can pull all in single before. Let me know about your opinion.

Reply:
Change done